src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <span class="rectangle"></span>
+    <header class="header">
+      <a href="index.html" class="logo"><img src="/images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item item_one"><a href="#" class="nav__link  is-active">Apple</a></li>
```

Comment:
Consider using the checklist to fix your code style accordingly to the style guide 
<img width="636" alt="Screenshot 2023-08-20 at 17 23 02" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/50458284-1fa6-4385-93d4-9d647933cc89">